src/arrayMethodSort.js

```diff
@@ -3,9 +3,40 @@
 /**
  * Implement method Sort
  */
+
+const compareStrings = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  } else if (strA === strB) {
+    return 0;
+  } else {
+    return -1;
+  }
+};
```

Comment:
You can use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)